e2e/src/collections.e2e-spec.ts

```diff
@@ -0,0 +1,31 @@
+import { AppPage } from './app.po';
+import { browser, by, element, protractor, logging } from 'protractor';
+
+describe('Collections', () => {
+    let page: AppPage;
+  
+    beforeEach(() => {
+      page = new AppPage();
+      //somehow need to update this to whatever should actually be appended to show collections
+      browser.getCurrentUrl().then(function (url) {
+        browser.get(url + "?showCollections=1");
+        });
+    });
+
+    it('should create a collection image', async () => {
+        expect
```

Comment:
this syntax seems to be causing the tests not to compile. If this test is failing you could comment it out to prevent it making the whole thing fail

Reply:
Also there are quite a few failures related to lint from this https://travis-ci.com/SubJunk/TimelineTools/jobs/210220962 that you can see in the Travis logs